src/App.vue

```diff
@@ -1,31 +1,39 @@
 <template>
   <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
-    <router-view/>
+    <app-header></app-header>
```

Comment:
if there's nothing inside tag you can omit the closing tag with slash in the end of first tag.
<app-header /> instead of
<app-header></app-header>